api/envoy/extensions/stat_sinks/open_telemetry/v3/open_telemetry.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package envoy.extensions.stat_sinks.open_telemetry.v3;
+
+import "envoy/config/core/v3/grpc_service.proto";
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.stat_sinks.open_telemetry.v3";
+option java_outer_classname = "OpenTelemetryProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/stat_sinks/open_telemetry/v3;open_telemetryv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Open Telemetry Stats Sink]
+// Stats configuration proto schema for ``envoy.stat_sinks.open_telemetry`` sink.
+// [#extension: envoy.stat_sinks.open_telemetry]
+
+// [#next-free-field: 10]
+message SinkConfig {
+  oneof protocol_specifier {
+    option (validate.required) = true;
+
+    // The upstream gRPC cluster that implements the OTLP/gRPC collector.
+    config.core.v3.GrpcService grpc_service = 1 [(validate.rules).message = {required: true}];
+
+    // The upstream cluster that implements the OTLP/HTTP collector.
+    // [#not-implemented-hide:]
+    string http_service = 2;
```

Comment:
As these are new protos, can you just remove all the not-implemented hide stuff? They can be added back later when the code is added, if it ever is. Same below.

Reply:
done